ivadomed/scripts/automate_training.py

```diff
@@ -407,19 +768,18 @@ def main(args=None):
     parser = get_parser()
     args = imed_utils.get_arguments(parser, args)
```

Comment:
When not entering the right flag (e.g., I used -p instead of -ch), the message error is not informative: 
```    raise ArgParseException('Error parsing args')
ivadomed.utils.ArgParseException: Error parsing args
```
vs. when using `parser.parse_args()`:
```
usage: automate_training.py [-h] -c <file> -ch <file> [-pd <int>] [-n <int>]
                            [--all-combin] [-m] [--run-test] [--fixed-split]
                            [-l] [-t <float>] [-o OUTPUT_DIR]
automate_training.py: error: the following arguments are required: -ch/--config-hyper
```

why not directly using parser.parse_args? We should probably fix this new function `imed_utils.get_argments`, but what is its purpose?

Reply:
Oh interesting! So the `imed_utils.get_arguments` function was implemented to allow for CLI testing, or to call the main function with a set of arguments. Basically it handles the case where you pass in the arguments instead of getting them from the command line. I agree though, this isn't a very good error message. I can open a separate issue to fix this?

Reply:
https://github.com/ivadomed/ivadomed/issues/669

Reply:
Sounds great, thank you!